src/screens/FeedScreen/index.js

```diff
@@ -230,7 +229,7 @@ const FeedScreen = (props) => {
   };
 
   return (
-    <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
+    <View>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `SafeAreaProvider` to `View` might cause layout issues on devices with notches or rounded corners. The `SafeAreaProvider` ensures that the content is displayed within the safe area boundaries of the device. If there's no specific reason for this change, I would suggest reverting it.
```diff
- <View>
+ <SafeAreaProvider style={styles.container} forceInset={{top: 'always'}}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->